index.ts

```diff
@@ -176,30 +189,21 @@ wss.on("request", function (request) {
     };
   }
   const room: Room = rooms[roomId];
+  let conn: Connection = {
```

Comment:
Having `let connection` and `let conn` here in the same scope is confusing. We should keep that in mind and maybe refactor this later a bit better.

Reply:
How about using the term `session` for the object which contains `id`, `name` and the Websocket `connection` object?